contracts/SnappAuction.sol

```diff
@@ -56,9 +75,61 @@ contract SnappAuction is SnappBase {
         return auctions[slot].appliedAccountStateIndex != 0;
     }
 
+    function getStandingOrderHash(uint16 userId, uint128 nonce) public view returns (bytes32) {
+        return reservedAccountOrders[userId][nonce].orderHash;
+    }
+    
+    function getStandingOrderValidityFrom(uint16 userId, uint128 nonce) public view returns (uint) {
+        return reservedAccountOrders[userId][nonce].validityFrom;
+    }
+
+    function getStandingOrderValidityTo(uint16 userId, uint128 nonce) public view returns (uint) {
+        return reservedAccountOrders[userId][nonce].validityTo;
+    }
+
     /**
      * Auction Functionality
      */
+    function placeStandingSellOrder(
+        uint8[] memory buyToken,
+        uint8[] memory sellToken,
+        uint128[] memory buyAmount,
+        uint128[] memory sellAmount
+    ) public onlyRegistered() {
```

Comment:
We might need to be stronger than only registered here... Maybe `onlySpecialAccount`.